core/cat/looking_glass/cheshire_cat.py

```diff
@@ -174,6 +175,15 @@ def store_new_message_in_working_memory(self, user_message_json):
         self.working_memory["user_message_json"]["prompt_settings"] = prompt_settings
 
 
+    def get_base_url(self):
+        return os.getenv("CORE_HOST")
```

Comment:
Please also use `CORE_SECURE_PROTOCOLS` and `CORE_PORT`
Example [here](https://github.com/cheshire-cat-ai/core/blob/main/core/cat/log.py#L167)

Reply:
fixed